### PR TITLE
feat: support auto imports

### PR DESCRIPTION
**Pull request by wattanx**
🔗 Linked issue
Fixes: https://github.com/wattanx/nuxt-pandacss/issues/38

❓ Type of change
- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

📚 Description
Enable auto imports of utilities provided by Panda CSS.

## TODO
- [x] patterns
- [x] auto imports option in Module Option

### CODE REVIEW

File: src/imports/presets.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import type { ModuleOptions } from "./../types";
+import type { InlinePreset } from "unimport";
+import { defineUnimportPreset } from "unimport";
+
+export const createPresets = ({
```
Comment (shinGangan) [inline src/imports/presets.ts:5]: ### Question

Is [`${outdir}/patterns`](https://panda-css.com/docs/concepts/patterns) not eligible for Auto-imports?

Comment (wattanx) [reply src/imports/presets.ts:5]: We will also support `patttern`!
(It is listed in the todo of the PR.)

Comment (shinGangan) [reply src/imports/presets.ts:5]: Oh, sorry. I overlooked it🙏

Comment (shinGangan) [reply src/imports/presets.ts:5]: I was stuck at work…
 I really appreciate you making the PR 🙏

**Comment by wattanx** [comment]
patterns and jsx are not working for the life of me, so I'll deal with it in a separate issue.

**Pull request merged**